docs/installation.md

```diff
@@ -2,422 +2,439 @@
+```
 
-## Installation
 
-```sh
-sudo cpanm Zonemaster::WebBackend
-```
+## 5. Configuration 
 
+There are three steps in this section:
 
-## Configuration
+1. Directory and file manipulation
+2. Set up service scripts for the relevant OS
+3. Database configuration and creation
 
-The Zonemaster::WebBackend module installs a number of configuration files in a shared data directory.
-This section refers to the shared data directory as the current directory, so locate it and go there like this:
 
-```sh
-cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-WebBackend")'`
+### 5.1 Directory and file manipulation
+
+```sh 
+sudo mkdir /etc/zonemaster 
+mkdir "$HOME/logs" 
 ```
 
-Create directories for configuration and log files.
+The Zonemaster::WebBackend module installs a number of configuration files in a
+shared data directory.  This section refers to the shared data directory as the
+current directory, so locate it and go there like this:
 
-```sh
-sudo mkdir /etc/zonemaster
-mkdir "$HOME/logs"
+```sh 
+cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-WebBackend")'` 
 ```
 
 Copy the `backend_config.ini` file to `/etc/zonemaster`.
 
-```sh
-sudo cp ./backend_config.ini /etc/zonemaster/
+```sh 
+sudo cp ./backend_config.ini /etc/zonemaster/ 
 ```
 
-Install service scripts for the relevant operating system:
-
-* CentOS
+### 5.2.1 CentOS
 
-  Copy the example init file to the system directory.
-  You may wish to edit the file in order to use a more suitable user and group.
-  As distributed, it uses the MySQL user and group, since we can be sure that exists and it shouldn't mess up anything included with the system.
+Copy the example init file to the system directory.  You may wish to edit the
+file in order to use a more suitable user and group.  As distributed, it uses
+the MySQL user and group, since we can be sure that exists and it shouldn't mess
+up anything included with the system.
 
-  ```sh
-  sudo cp ./zm-centos.sh /etc/init.d/
-  sudo chmod +x /etc/init.d/zm-centos.sh
-  ```
+```sh 
+sudo cp ./zm-centos.sh /etc/init.d/ 
+sudo chmod +x /etc/init.d/zm-centos.sh
+```
 
-* Debian and Ubuntu
+### 5.2.2 Debian and Ubuntu
 
-  Copy the file `./zm-backend.sh` to the directory `/etc/init`, make it an executable file, and add the file to start up script.
+Copy the file `./zm-backend.sh` to the directory `/etc/init`, make it an
+executable file, and add the file to start up script.
 
-  ```sh
-  sudo cp ./zm-backend.sh /etc/init.d/
-  sudo chmod +x /etc/init.d/zm-backend.sh
-  sudo update-rc.d zm-backend.sh defaults
-  ```
+```sh 
+sudo cp ./zm-backend.sh /etc/init.d/ 
+sudo chmod +x /etc/init.d/zm-backend.sh 
+sudo update-rc.d zm-backend.sh defaults 
+```
 
-  >
-  > At this time there is no instruction for running Zonemaster *Workers* as services on Debian and Ubuntu.
-  >
+>
+> At this time there is no instruction for running Zonemaster *Workers* as
+> services on Debian and Ubuntu.
+>

```

Comment:
The zm-backend.sh script will start both the Web Backend and the Worker as daemons.